### PR TITLE
Translated some strings to Italian

### PR DESCRIPTION
**Pull request by blackgin**

### CODE REVIEW

File: app/src/main/res/values-it/strings.xml [hunk 1]
```diff
@@ -106,7 +112,14 @@
     <string name="dark_theme">Tema scuro</string>
     <string name="pref_start_screen">Schermata iniziale</string>
```
Comment (paronos) [inline app/src/main/res/values-it/strings.xml]: <string name="pref_language">Language</string>

Comment (paronos) [inline app/src/main/res/values-it/strings.xml]: `<string name="pref_language">Lingua</string>`

File: app/src/main/res/values-it/strings.xml [hunk 2]
```diff
@@ -106,7 +112,14 @@
...
 
-      <!-- Reader section -->
+    <!-- Languages -->
+    <string name="system_default">Predefinito di sistema</string>
```
Comment (paronos) [inline app/src/main/res/values-it/strings.xml]: Predefinit**a** di sistema

File: app/src/main/res/values-it/strings.xml [hunk 3]
```diff
@@ -164,10 +177,10 @@
     <string name="fourth_to_last">Quartultimo capitolo</string>
     <string name="fifth_to_last">Quintultimo capitolo</string>
```
Comment (paronos) [inline app/src/main/res/values-it/strings.xml]: `<string name="pref_download_new">Scarica nuovi capitoli</string>`

File: app/src/main/res/values-it/strings.xml [hunk 4]
```diff
@@ -211,6 +224,9 @@
     <string name="select_source">Selezionare una fonte</string>
     <string name="no_valid_sources">Per favore abilita almeno una fonte valida</string>
 
+    <!-- Manga activity -->
+    <string name="manga_not_in_db">Questo manca è stato rimosso dal database</string>
```
Comment (paronos) [inline app/src/main/res/values-it/strings.xml]: Questo mang**a**

**Comment by inorichi** [comment]
I think we shouldn't translate languages, but hardcode them in their respective languages. Should I merge this or is anything going to be changed?

**Comment by blackgin** [comment]
You can merge, unless you want me to update the files according to your comment

**Pull request merged**